scss/mixins/_grid.scss

```diff
@@ -29,7 +29,7 @@
 @mixin make-col($size: false, $columns: $grid-columns) {
   @if $size {
     flex: 0 0 auto;
-    width: percentage(divide($size, $columns));
+    width: percentage(divide($size, $columns, 12));
```

Comment:
Using 12 here doesn't seem to change any of the output with Dart Sass (when testing quickly on Sassmeister, at least), likely because Dart Sass has a fixed precision of 10. This value of 12 versus [our docs](https://getbootstrap.com/docs/5.0/getting-started/build-tools/#sass) recommending 6 as the minimum (to avoid browser rounding errors) could lead to unnecessary confusion for reading and/or maintaining this later. I think we should either use 6 as we recommend in the docs, or just exclude the parameter so it uses the `divides`'s default of 10, which also happens to match up with Dart Sass's "overkill" fixed precision of 10.

Reply:
If you wouldn't mind, please consider trying locally. The default value fo 10 showed these values as 2 decimal places too short for me, hence the 12 in this situation.

/cc @alpadev

Reply:
I guess that SASS internally uses more precise numbers and is rounding them to a fixed length when compiling. The result for `8 / 12` is the same as `divide(8, 12)`. When the result is multiplied e.g. `* 100%` the results vary. I can't really do something here..

Reply:
We could change it like `divide(100%, divide(12, 8))` so there would be no need to change the parameter (in that case we'd have to add `'%': 1%` to the `$unit-map`).